res/values/strings_en.arb

```diff
@@ -322,6 +322,7 @@
   "error_text_address" : "Wallet address must correspond to the type\nof cryptocurrency",
   "error_text_node_address" : "Please enter a iPv4 address",
   "error_text_node_port" : "Node port can only contain numbers between 0 and 65535",
+  "error_text_node_proxy_address" : "Please enter <iPv4 address : port>",
```

Comment:
Minor: please change to `Please enter <IPv4 address>:<port>, for example 127.0.0.1:9050` and localize

Reply:
done